lib/service/database.js

```diff
@@ -87,7 +87,9 @@ DatabaseService.prototype.getMetadataFromAffectedTables = function(node, skip, c
         var defaultLastUpdatedTime = new Date('1970-01-01T00:00:00.000Z');
         var affectedTables = [];
         var lastUpdatedTimes = [];
-        var rows = resultSet.rows || [defaultLastUpdatedTime, null, null];
+        var rows = (Array.isArray(resultSet.rows) && resultSet.rows.length > 0) ?
+            resultSet.rows :
+            [defaultLastUpdatedTime, null, null];
```

Comment:
That's something I added but it looks wrong. It should be `[{"updated_at": defaultLastUpdatedTime, "schema_name": null, "table_name": null}]`